src/Libraries/OrchardCore.Commerce.Abstractions/Abstractions/IShoppingCartPersistence.cs

```diff
@@ -8,7 +7,7 @@ namespace OrchardCore.Commerce.Abstractions;
 /// Service that provides a way to retain shopping cart information.
 /// </summary>
 /// <remarks><para>
-/// When deriving a custom implementation, please inherit from <see cref="ShoppingCartPersistenceBase"/> to retain event
+/// When deriving a custom implementation, please inherit from ShoppingCartPersistenceBase to retain event
```

Comment:
What's the reason of this change?

Reply:
You've marked this as resolved without answering the question. Please resolve conversations, that's for the reviewer. (Automatically resolving via "commit suggestion" is fine.)

Reply:
You've marked this as resolved without answering the question. Please resolve conversations, that's for the reviewer. (Automatically resolving via "commit suggestion" is fine.)